modules/subscriptions/subscriptions/langs/en.json

```diff
@@ -40,6 +40,7 @@
   "invoice-history": {
     "invoice-id-th": "Invoice Id",
     "date-th" : "Date",
+    "validity-th" : "Valid until",
     "amount-th" : "Amount",
     "action-th" : "Actions",
     "total-label" : "Total Records:",
```

Comment:
is there any specific reason that we have : behind Total Records?

Reply:
`Total Records: 1 of 3`